its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -272,7 +272,7 @@ private static Path getNuGetPath(Orchestrator orch) {
     return nugetPath;
   }
 
-  private static BuildResult runMSBuildQuietly(Orchestrator orch, Path projectDir, List<EnvironmentVariable> environmentVariables, String... arguments) {
+  public static BuildResult runMSBuildQuietly(Orchestrator orch, Path projectDir, List<EnvironmentVariable> environmentVariables, String... arguments) {
```

Comment:
note: the layout of the whole file is a bit of a mess (private, public methods are all around the place and not grouped by visibility), but I don't think this IT code is the best use of our time in changing the structure (public , then private methods)